### PR TITLE
Update/to diesel 2.0

### PR DESCRIPTION
**Pull request by weiznich**

### CODE REVIEW

**Comment by geomaniac** [comment]
Very hard to review in detail, having not looked at this since before the introduction of the `oracle` crate and without in-depth knowledge of diesel internals. Is there anything particular someone should have a look at?

File: src/oracle/connection/mod.rs [hunk 1]
```diff
@@ -164,56 +166,33 @@ impl Connection for OciConnection {
         }
         url += path;
 
-        let mut raw = oracle::Connection::connect(user, password, url)
+        let mut raw = oracle::Connection::connect(dbg!(user), dbg!(password), dbg!(url))
```
Comment (weiznich) [inline src/oracle/connection/mod.rs:169]:
```suggestion
        let mut raw = oracle::Connection::connect(user, password, url)
```

**Comment by weiznich** [comment]
>  Is there anything particular someone should have a look at?

In my option just a general light review should be fine. So just look if something looks strange/broken. If that's not the case we should just run the full gst-test suite after updating our service and see if something breaks there.

File: src/oracle/connection/oracle_value.rs [hunk 2]
```diff
@@ -9,12 +9,11 @@ pub struct OracleValue<'a> {
 pub(crate) enum InnerValue<'a> {
     Raw {
         raw_value: &'a oracle::SqlValue,
-        tpe: &'a oracle::sql_type::OracleType,
+        tpe: oracle::sql_type::OracleType,
     },
     SmallInt(i16),
     Integer(i32),
     BigInt(i64),
-    Bool(bool),
```
Comment (MrMuetze) [inline src/oracle/connection/oracle_value.rs:17]: Is it ok that `bool` is gone now?

Comment (weiznich) [reply src/oracle/connection/oracle_value.rs:17]: It is not needed anymore as we use `SmallInt` internally for this now (https://github.com/GiGainfosystems/diesel-oci/pull/41/files/2748b78bc2148be11b39d0699ebe25d2869f0cd9#diff-e6df06b79f25b6537cfe851fe8c707084197677482e0edf94f74655b4baa3f5aR359) and handle the conversion directly in the corresponding `FromSql` impl: https://github.com/GiGainfosystems/diesel-oci/pull/41#diff-08866ee1105680dbb2a25006ffd51ffb5b98b7358c1752d2cbbec40a0fa5d4eaR169

**Pull request merged**